doc/articles/faq.md

```diff
@@ -8,7 +8,7 @@ uid: Uno.Development.FAQ
 
 #### What is the Uno Platform?
 
-Uno Platform is a cross-platform application framework which lets you write an application once in XAML and C#, and deploy it to [any target platform](getting-started/requirements.md). Read more [here](what-is-uno.md).
+Uno Platform is a cross-platform application framework which lets you write an application once in XAML and/or C#, and deploy it to [any target platform](getting-started/requirements.md). Read more [here](what-is-uno.md).
```

Comment:
Platform vs. Framework. Uno Platform is a cross-platform application framework ..doesn't sound right. 

I suggest 
Uno Platform lets you write....